### PR TITLE
Weekly-Updates for Codec-Registry 

### PR DESCRIPTION
**Pull request by Alt-Shivam**

### CODE REVIEW

**Comment by Alt-Shivam** [comment]
Will add more commits soon.
@MSanKeys963

File: 2022/codec_registry_shivank/README.md [hunk 1]
```diff
@@ -1 +1,93 @@
...
+## Related links:
+1) https://github.com/zarr-developers/numcodecs
+2) https://github.com/zarr-developers/numcodecs/issues/278
+3) https://support.hdfgroup.org/services/filters.html
+4) https://github.com/cgohlke/imagecodecs
+
+# Project Weekly Updates:
+
+## Pre-GSoC Work
+* Created a Research file.
```
Comment (sanketverma1704) [inline 2022/codec_registry_shivank/README.md]: Can you please add link to your HackMd research file here too? Thanks!

File: 2022/codec_registry_shivank/README.md [hunk 2]
```diff
@@ -1 +1,93 @@
...
+Astype
+Bitround
+JSON
+```
+* Documented Codec informations in Research file.
+* Created a Pull Request [Link](https://github.com/zarr-developers/zarr-python/pull/1016)
+
+## Week 3 & 4
+* Created a Github Repo to keep registry files. [Link](https://github.com/Alt-Shivam/Codecs-Registry)
+* Created a **csv** file for Registry.
```
Comment (sanketverma1704) [inline 2022/codec_registry_shivank/README.md]: Please hyperlink `.csv` file here.

File: 2022/codec_registry_shivank/README.md [hunk 3]
```diff
@@ -1 +1,93 @@
...
+and there documentation in research file.
+* Created a Pull Request. [Link](https://github.com/zarr-developers/zarr-python/pull/1047)
+
+## Week 5 & 6
+* Created a Github-Pages website to show Codec Registry. [Link](https://alt-shivam.github.io/Codecs-Registry/)
+* Shifted Codec research file from docs to .md.
+* Created a saperate page to show the research file. [Link](https://alt-shivam.github.io/Codecs-Registry/Others/Research.html)
+* Added Basic Information on CodecRegistry website.
+
+# Mid-Term Evaluations. :tada:
```
Comment (sanketverma1704) [inline 2022/codec_registry_shivank/README.md:82]: If you want to add something here to show that you've passed the mid-term evaluation, that would be great. Maybe a screenshot from GSoC 2022 portal?

**Comment by sanketverma1704** [comment]
Hi @Alt-Shivam, thanks for sending this PR.
Apart from a few comments, it LGTM. Let me know when you fix them, and I'll merge this.

Also, you can add [this ](https://hackmd.io/@uTe8Vo8gSYeCbwHsQI2Z2Q/SypXtPRD9) somewhere at the top of the README.md.

File: 2022/codec_registry_shivank/README.md (hunk 1 above)
Comment (Alt-Shivam) [reply 2022/codec_registry_shivank/README.md]: Sure

File: 2022/codec_registry_shivank/README.md (hunk 2 above)
Comment (Alt-Shivam) [reply 2022/codec_registry_shivank/README.md]: Sure, 🎉

File: 2022/codec_registry_shivank/README.md (hunk 3 above)
Comment (Alt-Shivam) [reply 2022/codec_registry_shivank/README.md:82]: Great Idea, Thanks

File: 2022/codec_registry_shivank/README.md [hunk 4]
```diff
@@ -1 +1,100 @@
...
+**Project length:** 175 hrs
+
+## Related links:
+1) https://github.com/zarr-developers/numcodecs
+2) https://github.com/zarr-developers/numcodecs/issues/278
+3) https://support.hdfgroup.org/services/filters.html
+4) https://github.com/cgohlke/imagecodecs
+
+# Project Weekly Updates:
+Project Progress and Information: [Link](https://hackmd.io/@uTe8Vo8gSYeCbwHsQI2Z2Q/SypXtPRD9)
```
Comment (MSanKeys963) [inline 2022/codec_registry_shivank/README.md:30]: This needs to be fixed.

**Pull request merged**